### ISSUE TITLE
loss exploding after some iterations

### ISSUE THREAD

**Issue by waminos** [kwuking/TimeMixer#5]
I use a custom dataset and train the model, but after some iterations the loss explode. 
![image](https://github.com/kwuking/TimeMixer/assets/18538948/f9e24005-24fd-4d2e-ad1f-673d2964b0e3)

How to solve this issue, or what could be the root cause ?

`!python -u run.py 
  --task_name long_term_forecast 
  --is_training 1 
  --freq t 
  --model TimesNet 
  --target OT 
  --model_id ECL_120'_'96 
  --model TimeMixer 
  --data custom 
  --features M 
  --seq_len 96 
  --label_len 0 
  --pred_len 96 
  --moving_avg 15 
  --e_layers 3 
  --d_layers 1 
  --enc_in 6 
  --dec_in 6 
  --c_out 6 
  --des 'Exp' 
  --itr 1 
  --d_model 32 
  --d_ff 64 
  --batch_size 8 
  --seasonal_patterns Daily 
  --learning_rate 0.01 
  --train_epochs 1 
  --patience 3 
  --down_sampling_layers 3 
  --down_sampling_method avg 
  --down_sampling_window 2

**Comment by kwuking** [comment]
> I use a custom dataset and train the model, but after some iterations the loss explode. ![image](https://private-user-images.githubusercontent.com/18538948/318271342-f9e24005-24fd-4d2e-ad1f-673d2964b0e3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE5NTA4MTEsIm5iZiI6MTcxMTk1MDUxMSwicGF0aCI6Ii8xODUzODk0OC8zMTgyNzEzNDItZjllMjQwMDUtMjRmZC00ZDJlLWFkMWYtNjczZDI5NjRiMGUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDAxVDA1NDgzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVhZWRmMDdlNmU1ZTFlYTZjMDZhOGZiNDY3ODY4OWM0YjBkYWEzYjFiMjQ5YTM5NDc0MGM3ZDg5NjBiZTllODEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.F-4vi6qiaCrdPvrGXSVrSVfsl8ggNJR7ihVQ_NN5g5k)
> 
> How to solve this issue, or what could be the root cause ?
> 
> `!python -u run.py --task_name long_term_forecast --is_training 1 --freq t --model TimesNet --target OT --model_id ECL_120'_'96 --model TimeMixer --data custom --features M --seq_len 96 --label_len 0 --pred_len 96 --moving_avg 15 --e_layers 3 --d_layers 1 --enc_in 6 --dec_in 6 --c_out 6 --des 'Exp' --itr 1 --d_model 32 --d_ff 64 --batch_size 8 --seasonal_patterns Daily --learning_rate 0.01 --train_epochs 1 --patience 3 --down_sampling_layers 3 --down_sampling_method avg --down_sampling_window 2

hi，the reason for this is that the learning_rate is too high. As shown above, the learning_rate is set to 0.01. You can solve this problem by adjusting the learning_rate appropriately.
